packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -319,10 +328,10 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
         this.intervalTree.map(fn);
     }
 
-    public createLegacyId(start: number, end: number): string {
+    public createLegacyId(): string {
         // Create a non-unique ID based on start and end to be used on intervals that come from legacy clients
         // without ID's.
-        return `${start}-${end}`;
+        return `legacy`;
     }
```

Comment:
don't these need to be globally unique?

Reply:
I don't think we can guarantee global uniqueness of start-end-based ID's, because, as you pointed out, intervals can change their (start, end), and then a new legacy interval with the old (start, end) can appear. The other situation I wanted to avoid is the one where you have multiple legacy intervals with the same (start, end) but different ID's, because one or more of them started out with a different (start, end). I don't know whether that's possible, but if it is, I think we're better off with a known ID that causes all the legacy intervals with the same (start, end) to collapse into one. Then delete(start, end) can just remove that one.

Reply:
If we do go with this change, should it go into release 42?